managers/views.py

```diff
@@ -34,3 +35,12 @@ def get(self, request):
         return Response({
             'projects_queryset': projects_queryset,
         })
+
+
+class ProjectDetail(APIView):
+    renderer_classes = [renderers.TemplateHTMLRenderer]
+    template_name = 'managers/project_detail.html'
```

Comment:
OK I think this is what we discussed on last meeting about Sheetstorm code review, but we couldn't get common understanding. So you want to use endpoints but actually return generated html code - is that correct? If yes, then are you planning on replacing whole html code with the received response?